frontend/src/app/forms/steps/education-info-entry/education-info-entry.component.html

```diff
@@ -0,0 +1,83 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button (click)="cancel()">Cancel</ion-button>
+    </ion-buttons>
+    <ion-buttons slot="end">
+      <ion-button [disabled]="!education.valid" (click)="save()"
+        >Save</ion-button
+      >
+    </ion-buttons>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <div class="main-container with-bottom-buttons">
+    <ion-list>
+      <div>
+        <ion-icon item-left name="business-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label class="item item-input" position="floating" color="medium"
+            >School/University</ion-label
+          >
+          <ion-input type="text" [formControl]="education.controls.university">
+          </ion-input>
+        </ion-item>
+      </div>
+
+      <div>
+        <ion-icon item-left name="book-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label position="floating" color="medium"> Subject</ion-label>
+          <ion-input
+            type="text"
+            [formControl]="education.controls.subject"
+          ></ion-input>
+        </ion-item>
+      </div>
+
+      <div>
+        <ion-icon item-left name="school-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label position="floating" color="medium">Degree</ion-label
+          ><ion-select
+            interface="popover"
+            [formControl]="education.controls.degree"
+          >
+            <ion-select-option
+              *ngFor="let degree of degreeTypeArray"
+              [value]="degree.value"
+            >
+              {{ degree.displayName }}</ion-select-option
+            >
+          </ion-select>
+        </ion-item>
+      </div>
+
+      <div>
+        <ion-icon item-left name="ribbon-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label position="floating" color="medium">Grade</ion-label
+          ><ion-input
+            type="number"
+            [formControl]="education.controls.grade"
+          ></ion-input>
+        </ion-item>
+      </div>
+
+      <div>
+        <ion-icon item-left name="time-outline" size="large"></ion-icon>
+        <ion-item style="display: inline-block;">
+          <ion-label position="floating" color="medium"
+            >Graduation Date</ion-label
+          ><ion-datetime
+            displayFormat="YYYY"
+            max="2030"
+            min="2000"
```

Comment:
Maybe, the minimum is a bit too strict :) How about people who have graduated 25 years ago?

Reply:
Not sure if people graduated 25 years ago will be attending job fair. Anyways will make min="1980" which could be enough.